charts/amazon-cloudwatch-observability/templates/linux/fluent-bit-configmap.yaml

```diff
@@ -14,8 +14,20 @@ data:
     {{- end }}
   parsers.conf: |
     {{- .Values.containerLogs.fluentBit.config.customParsers  | nindent 4 }}
+{{- if contains "us-iso-" .Values.region }}
```

Comment:
nit: Could use `hasPrefix` instead https://helm.sh/docs/chart_template_guide/function_list/#hasprefix-and-hassuffix